landscape/client/manager/tests/test_ubuntuproinfo.py

```diff
@@ -97,6 +112,71 @@ def test_ubuntu_pro_info(self):
         result = json.loads(messages[0]["ubuntu-pro-info"])
         self.assertTrue(result["attached"])
 
+    def test_serializable(self):
+        """Tests calling `pro status` when it is not installed."""
```

Comment:
This is a bit of nit, you don't have to do it. I prefer tests to mention what they test and what they _expect_. Here this would change to _Calling `pro status` when it is not installed should do X_. The description is probably wrong anyway.